packages/combobox/src/Combobox.ts

```diff
@@ -211,6 +214,13 @@ export class Combobox extends Textfield {
             return;
         }
         this.value = this.activeDescendant.itemText;
+
+        const activeEl = this.shadowRoot.getElementById(
+            this.activeDescendant.value
+        );
+        if (activeEl) {
+            activeEl.click();
+        }
```

Comment:
Does this change also trigger the code in line 216 to happen in `handleInput`, as well?

Reply:
Ah indeed.
It does not trigger `handleInput`, buuuut it does trigger `handleMenuChange` which does `this.value = selected?.itemText || '';` which is same thing as line 216.